cdap-common/src/main/java/co/cask/cdap/common/lang/FilterClassLoader.java

```diff
@@ -97,7 +126,7 @@ protected Package getPackage(String name) {
 
   @Override
   public URL getResource(String name) {
-    return super.getResource(name);
+    return resourceAcceptor.apply(name) ? super.getResource(name) : null;
```

Comment:
You have the override `getResources` and `getResoruceAsStream` as well. Also, remove `findResource` and `findResources`.